aptos-move/move-examples/on_chain_dice/sources/dice.move

```diff
@@ -8,15 +8,15 @@ module module_owner::dice {
     }
 
     #[randomness]
-    entry fun roll(account: signer) acquires DiceRollHistory {
-        let addr = address_of(&account);
+    public entry fun roll(account: &signer) acquires DiceRollHistory {
```

Comment:
i think this has to remain private, otherwise randomness API call won't work.

Reply:
you can create transactions with private entry functions.

private is just for other call referring to it
entry is whether transaction can call it or not

Reply:
I mean randomness txns can’t start with a public function, otherwise malicious users can test-and-abort.

we also made vm changes to ensure it, and this change should have failed a smoke-test.

Reply:
I was saying to Satya that he can drop public and still call it, he doesn't need "public" here at all

Reply:
Changed the function to private.